source/_components/xbox_live.markdown

```diff
@@ -14,7 +14,11 @@ The Xbox Live integration is able to track [Xbox](http://xbox.com/) profiles.
 
 To use this sensor you need a free API key from
 [XboxAPI.com](http://xboxapi.com).
-Please also make sure to connect your Xbox account on that site.
+Please also make sure to connect your Xbox account on that site. As of 
```

Comment:
Isn't it better to raise an issue to get this fixed?

Reply:
To raise an issue to get "what" fixed? Do you mean to implement a "custom" scan interval within the xbox_live component?

Reply:
Change the defaults of the integration in the code base.

Reply:
I'm new to this, so I was going with a "light touch" on making changes. Also, this is only an issue for users of the _unpaid_ API, so it's not a universal problem. I also figure it's easier to change documentation than code. I bow to your expertise, though. Please let me know if I should open an issue for the _underlying_ "problem". Thanks again!